src/proto/single/resp.rs

```diff
@@ -118,6 +120,92 @@ pub async fn read_ok<R: AsyncBufRead + Unpin>(r: R) -> Result<(), Error> {
     Err(bad("server ok", &rr).into())
 }
 
+// ----------------------------------------------
+
+/// Faktory service stats.
+#[derive(Serialize, Deserialize, Debug, Clone)]
```

Comment:
Any particular reason to also implement `Serialize` here, or can we remove that?

Reply:
As a user, I actually would expect most of the things to be `Serialize` for me to be able to send them further

Reply:
wdyt?

Reply:
I suppose for forwarding, or maybe also for mocking. Okay, fair enough! :+1: